src/firefly/js/api/ApiUtil.js

```diff
@@ -42,15 +49,11 @@ export function debug(...msg) {
 
 
 
-
-
-
 /**
  *
  * @param {string|Object} div a div element or a string id of the div element
  * @param {Object} Component a react component
  * @param {Object} [props] props for the react component
- * @namespace firefly
  * @Function renderDOM
```

Comment:
Yes, it is not necessary.  There ware still try/error things left in the source codes.  Usually the @function is not needed.